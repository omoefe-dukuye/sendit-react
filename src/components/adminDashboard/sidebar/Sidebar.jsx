import React from 'react';
import PropTypes from 'prop-types';
import { NavLink } from 'react-router-dom';
import './styles.scss';

const SideBar = () => {
  return (
    <div className='sidebar'>
      <div className='sidebar-links'>
        <ul>
          <li className='sidebar-links__link'>
            <NavLink to='/admin-dashboard/status' activeClassName='sidebar-links__link--active'>Status</NavLink>
          </li>
          <li className='sidebar-links__link'>
            <NavLink to='/admin-dashboard/location' activeClassName='sidebar-links__link--active'>Location</NavLink>
          </li>
          <li className='sidebar-links__link'>
            <NavLink to='/admin-dashboard/details' activeClassName='sidebar-links__link--active'>Details</NavLink>
          </li>
          <li className='sidebar-links__link'>
            <NavLink to='/admin-dashboard/user' activeClassName='sidebar-links__link--active'>User</NavLink>
          </li>
          <li className='sidebar-links__link'>
            <NavLink to='/admin-dashboard/all' activeClassName='sidebar-links__link--active'>All</NavLink>
          </li>
        </ul>
      </div>
    </div>
  );
};

SideBar.propTypes = {
  authUserEmail: PropTypes.string
};

export default SideBar;
