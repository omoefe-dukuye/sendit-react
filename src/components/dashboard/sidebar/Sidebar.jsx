import React from 'react';
import PropTypes from 'prop-types';
import { NavLink } from 'react-router-dom';
import './styles.scss';

const SideBar = () => {
  return (
    <div className='sidebar'>
      <div className='sidebar-links'>
        <ul>
          <li className='sidebar-links__link'>
            <NavLink to='/dashboard/create' activeClassName='sidebar-links__link--active'>Create</NavLink>
          </li>
          <li className='sidebar-links__link'>
            <NavLink to='/dashboard/update' activeClassName='sidebar-links__link--active'>Update</NavLink>
          </li>
          <li className='sidebar-links__link'>
            <NavLink to='/dashboard/details' activeClassName='sidebar-links__link--active'>Details</NavLink>
          </li>
          <li className='sidebar-links__link'>
            <NavLink to='/dashboard/cancel' activeClassName='sidebar-links__link--active'>Cancel</NavLink>
          </li>
          <li className='sidebar-links__link'>
            <NavLink to='/dashboard/all' activeClassName='sidebar-links__link--active'>All</NavLink>
          </li>
        </ul>
      </div>
    </div>
  );
};

SideBar.propTypes = {
  authUserEmail: PropTypes.string
};

export default SideBar;
