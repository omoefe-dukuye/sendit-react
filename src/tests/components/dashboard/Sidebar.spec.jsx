import React from 'react';
import '@babel/polyfill';
import { shallow } from 'enzyme';
import Sidebar from '../../../components/adminDashboard/sidebar';

test('should render static component', () => {
  const wrapper = shallow(<Sidebar />);
  expect(wrapper).toMatchSnapshot();
});