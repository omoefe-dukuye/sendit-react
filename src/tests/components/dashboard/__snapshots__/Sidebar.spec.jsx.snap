// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render static component 1`] = `
<div
  className="sidebar"
>
  <div
    className="sidebar-links"
  >
    <ul>
      <li
        className="sidebar-links__link"
      >
        <NavLink
          activeClassName="sidebar-links__link--active"
          to="/admin-dashboard/status"
        >
          Status
        </NavLink>
      </li>
      <li
        className="sidebar-links__link"
      >
        <NavLink
          activeClassName="sidebar-links__link--active"
          to="/admin-dashboard/location"
        >
          Location
        </NavLink>
      </li>
      <li
        className="sidebar-links__link"
      >
        <NavLink
          activeClassName="sidebar-links__link--active"
          to="/admin-dashboard/details"
        >
          Details
        </NavLink>
      </li>
      <li
        className="sidebar-links__link"
      >
        <NavLink
          activeClassName="sidebar-links__link--active"
          to="/admin-dashboard/user"
        >
          User
        </NavLink>
      </li>
      <li
        className="sidebar-links__link"
      >
        <NavLink
          activeClassName="sidebar-links__link--active"
          to="/admin-dashboard/all"
        >
          All
        </NavLink>
      </li>
    </ul>
  </div>
</div>
`;
